do.py

```diff
@@ -26,6 +26,126 @@
     os.environ["PATH"], GO_BIN_PATH, GO_HOME_BIN_PATH, LOCAL_BIN_PATH
 )
 
+models_version = "0.11.6"
+sdk_version = "0.11.7"
```

Comment:
extract these from setup.py

Reply:
should move these values to a common json or something from where we can actually use in both the places